app/models/response_audit.rb

```diff
@@ -0,0 +1,9 @@
+class ResponseAudit < ApplicationRecord
+  belongs_to :request_audit
+
+  scope :person_reads, ->(person_id) do
+    direct_query = { id: person_id, type: 'people' }.to_json
+    relationship_query = { relationships: { person: { data: { id: person_id, type: 'people' } } } }.to_json
+    where('response @> ?', direct_query).or(where('response @> ?', relationship_query))
+  end
```

Comment:
This scope is quite complex, could we use 2 simpler different ones or simplify this one?

`direct_query`,  `relationship_query` are not very meaningful  (maybe something like  `person_id`, `person_relationship`, not sure anyway)

Reply:
I was looking for some good names, and these were the best I could find. Not sure how I could simplify this scope - it's just 2 straightforward queries joined by an 'or'. Open to suggestions.